tutorials/Tutorial15_TableQA.ipynb

```diff
@@ -132,10 +132,11 @@
     "from haystack.document_stores import ElasticsearchDocumentStore\n",
     "\n",
     "# We want to use a small model producing 512-dimensional embeddings, so we need to set embedding_dim to 512\n",
+    "document_index=\"document\"",
```

Comment:
I think we need a `,\n",` at the end of the line instead. Otherwise the markdown file looks bad with no space between that line and the next: `document_index="document"document_store`
Otherwise looks good to me! 👍

Reply:
Oh, I see it now. Thanks!